cmd/lady/server.go

```diff
@@ -64,7 +64,8 @@ func doServe(cc *cli.Context) {
 	http.Handle("/", http.FileServer(http.FS(contentStatic)))
 
 	port := int64(cc.Int(flagHttpPort))
-	err := http.ListenAndServe("localhost:"+strconv.FormatInt(port, 10), nil)
+
+	err := http.ListenAndServe(":"+strconv.FormatInt(port, 10), nil)
```

Comment:
we can just leave this change - I just changed it like this so that the server would be accessible as every possible host (localhost, 127.0.0.1, machine IP address)